tools/packaging/release/update-repository-version.sh

```diff
@@ -90,24 +118,26 @@ bump_repo() {
 
 	pushd "${repo}" >>/dev/null
 
+	local kata_deploy_dir="tools/packaging/kata-deploy"
+	local kata_deploy_base="${kata_deploy_dir}/kata-deploy/base"
+	local kata_cleanup_base="${kata_cleanup_dir}/kata-cleanup/base"
+	local kata_deploy_yaml="${kata_deploy_base}/kata-deploy.yaml"
+	local kata_cleanup_yaml="${kata_cleanup_base}/kata-cleanup.yaml"
+	local kata_deploy_stable_yaml="${kata_deploy_base}/kata-deploy-stable.yaml"
+	local kata_cleanup_stable_yaml="${kata_cleanup_base}/kata-cleanup-stable.yaml"
+
 	branch="${new_version}-branch-bump"
 	git fetch origin "${target_branch}"
 	git checkout "origin/${target_branch}" -b "${branch}"
 
-	# All repos we build should have a VERSION file
-	if [ ! -f "VERSION" ]; then
-		current_version="new"
-		echo "${new_version}" >VERSION
-	else
-		current_version="$(grep -v '#' ./VERSION)"
-
-		info "Updating VERSION file"
-		echo "${new_version}" >VERSION
-		if git diff --exit-code; then
-			info "${repo} already in version ${new_version}"
-			cat VERSION
-			return 0
-		fi
+	local current_version="$(grep -v '#' ./VERSION)"
```

Comment:
Nit: Might be worth anchoring this (`^#`) and potentially also excluding blank lines:

```suggestion
	local current_version="$(egrep -v '^(#|$)' ./VERSION)"
```